boards/arm/nucleo_wba52cg/support/openocd.cfg

```diff
@@ -0,0 +1,26 @@
+# Note: Using OpenOCD using nucloe_wba52cg requires using STMicroelectronics
+# openocd fork. See board documentation for more information
+
+source [find interface/stlink-dap.cfg]
+
+set WORKAREASIZE 0x8000
+
+transport select "dapdirect_swd"
+
+# Enable debug when in low power modes
+set ENABLE_LOW_POWER 1
+
+# Stop Watchdog counters when halt
+set STOP_WATCHDOG 1
+
+# STlink Debug clock frequency
+set CLOCK_FREQ 8000
+
+# Reset configuration
+# use hardware reset, connect under reset
+# connect_assert_srst needed if low power mode application running (WFI...)
+reset_config srst_only srst_nogate
+
+source [find target/stm32wbax.cfg]
+
+gdb_memory_map disable
```

Comment:
This should not be required

Reply:
@erwango Thanks, I've tested with it removed and it's ok :+1: